docs/about/index.md

```diff
@@ -4,18 +4,28 @@ title: About Amplication
 
 # About Amplication
 
-Amplication is a backend development platform that simplifies and speeds up the process of building Node.js apps.
+Amplication is a backend development platform that simplifies and accelerates the process of creating scalable and maintainable backend services. With support for both Node.js and .NET, Amplication provides developers with the flexibility to select the best technology stack for their needs.
 
-Amplication is open-source software, and it is free for all to use. Learn more about our [licensing model](/about/licensing). Also, we're committed to a policy of complete transparency about our future direction. Learn more about our [product roadmap](/about/roadmap).
+## Open Source and Commercial Offerings
+
+Amplication is proud to offer its Node.js application generation tools as open-source software and free for all to use. This ensures that developers can freely access, modify, and distribute the source code for their Node.js applications.
+
+The .NET functionality and Amplication's Enterprise Edition is available under a separate licensing model. For more information regarding the licensing details, please visit our [licensing model](/about/licensing) page.
+
+We are committed to a policy of complete transparency about our future direction. Learn more about what we have planned for Amplication and the various technologies we support by visiting our [product roadmap](/about/roadmap).
 
 ## How Amplication can help you
 
-Our main goal is to help Node.js developers build apps faster and more efficiently. To help you do that, here are some of the features that Amplication offers:
+Our main goal is to help developers build apps Node.js and .NET apps faster and more efficiently. To help you do that, here are some of the features that Amplication offers:
+
+- [**Generate Human-Readable Code**](/getting-started/): Amplication automatically generates fully functional, human-readable, and editable services. This includes applications based on TypeScript and Node.js, as well as robust, scalable applications using popular .NET frameworks.
```

Comment:
@mulygottlieb - Can you please assist here with "popular .NET frameworks"?

Reply:
@dericksozo the word "apps" returns twice here and the first should be removed: "Our main goal is to help developers build apps Node.js and .NET apps"

Reply:
@dericksozo like a previous comment from Paz, every place you use "build" for apps (like here) you should probably use "build and evolve" instead to emphasize the continuous value rather than a one-time value

Reply:
@PazYanoverr @dericksozo  I would not even say "popular .NET frameworks" since in .NET, the "framework" is mostly .NET itself, and since ".NET framework" is a really older version of what is called today just ".NET".
So I would rewrite the sentence as "This includes applications based on TypeScript and Node.js, as well as robust and scalable applications based on .NET"

Reply:
@PazYanoverr @dericksozo I saw that later on you use " popular .NET frameworks such as ASP.NET Core and Entity Framework Core." and this is actually a good starting point to use here too if you want.